.github/workflows/release_gem.yml

```
@@ -0,0 +1,14 @@
+name: Release
+
+on:
+  create:
+    ref_type: tag
```

```suggestion
  push:
    tags:
      - 'v*'
```

It's been said this is better (see https://github.com/theforeman/actions/?tab=readme-ov-file#gem-release)